src/components/common-components/netlify-pro-footer/netlify-pro-footer.component.tsx

```diff
@@ -12,8 +12,8 @@ const useStyles = makeStyles({
     background: ScssVars.white,
     padding: 10,
     borderTop: "1px solid #E9EEF9",
```

Comment:
Wiem, że to nie Twój kod, ale mógłbyś to też wrzucić do zmiennych?

Reply:
Tylko co? Bo jak dla mnie nie ma sensu robić zmiennych netlifyProFooterPadding i netlifyProFooterBorderTop?

Reply:
Już lepiej przenieść style poprostu do scss. Ja piszę w javascript style bo mi tak wygodniej i nie trzeba skakać po plikach, ale mogę pisać w scss

Reply:
Chodziło mi o ten kolor, żebyś my nie używali różnych

Reply:
A to już do Agi trzeba z tym iść